src/System.Management.Automation/help/HelpCommands.cs

```diff
@@ -612,12 +653,101 @@ private void WriteObjectsOrShowOnlineHelp(HelpInfo helpInfo, bool showFullHelp)
                             }
                         }
 
-                        WriteObject(helpInfo.ShortHelp);
+                        WriteObjectOrWritePaged(helpInfo.ShortHelp, bufferOutputWhenPaged: true);
                     }
                 }
             }
         }
 
+        private void WriteObjectOrWritePaged(object sendToPipeline, bool bufferOutputWhenPaged)
+        {
+            if (Paged)
+            {
+                if (pager is null)
+                {
+                    pager = new Pager();
+                }
+
+                var helpText = GetHelpOutput();
+
+                if (bufferOutputWhenPaged)
+                {
+                    multiItemPagerBuffer = helpText;
+                }
+                else
+                {
+                    pager.Write(helpText);
+                }
+            }
+            else
+            {
+                WriteObject(sendToPipeline);
+            }
+        }
+
+        private string GetHelpOutput()
+        {
+            string helpContentAsString = null;
+
+            using (var ps = System.Management.Automation.PowerShell.Create(RunspaceMode.CurrentRunspace))
+            {
```

Comment:
Could you elaborate? I am not aware of the new syntax.

Reply:
instead of:
```csharp
using (var ps = ... )
{
    ...
}
```

you can do:
```csharp
using var ps = ...;

...
```

No enclosing braces required. 🙂

Reply:
Yes, @vexx32  thanks for clarify.

Reply:
Thanks, yes found that and updated